dig/default.go

```diff
@@ -20,30 +20,30 @@
 
 package dig
 
-// Default graph is used for all the top-level inject calls
-var defaultGraph = newGraph()
+// DefaultGraph is used for all the top-level inject calls
+var DefaultGraph Graph = newGraph()
```

Comment:
I am little bit not sure about this change. Allowing other package has the ability to swap/overwrite global singleton, won't it introduce more chance of race? Like the one we saw in Parallel tests with global tracer. I am also worried some users don't know what they are doing gonna overwrite it by mistake and get into random runtime error.

I may need some help to understand why expose this variable make Parallel test easier as well...